### PR TITLE
[AzureStack][Admin API]  Adding disk list api in a new api version

### PR DESCRIPTION
**Pull request by FireDefend**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Please ensure to add changelog with this PR by answering the following questions.
  1. What's the purpose of the update?    
      - [ ] new service onboarding 
      - [ X] new API version 
      - [x] update existing version for new feature 
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify 
  2. When you are targeting to deploy new service/feature to public regions? Please provide date, or month to public if date is not available yet.
  3. When you expect to publish swagger? Please provide date, or month to public if date is not available yet.
  4. If it's an update to existing version,  please select SDKs of specific language and CLIs that require refresh after swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [X ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No, no need to refresh for updates in this PR

### Contribution checklist:
- [X] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [X] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [X] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  -  [x] Ensure to copy the existing version into new directory structure for first commit (including refactoring) and then push new changes including version updates in separate commits. This is required to review the changes efficiently.
  - Adding a new service

- [x] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/azsadmin/resource-manager/commerce/Microsoft.Commerce.Admin/preview/2015-06-01-preview/Commerce.json [hunk 1]
```diff
@@ -105,7 +105,7 @@
         "tags": [
           "Commerce"
         ],
-        "operationId": "UpdateEncryption",
+        "operationId": "Commerce_UpdateEncryption",
```
Comment (bganapa) [inline specification/azsadmin/resource-manager/commerce/Microsoft.Commerce.Admin/preview/2015-06-01-preview/Commerce.json:108]: pls remove this commit, there is a separate PR for this commit https://github.com/Azure/azure-rest-api-specs/pull/13283

File: specification/azsadmin/resource-manager/compute/Microsoft.Compute.Admin/stable/2021-04-01/Disks.json [hunk 2]
```diff
@@ -0,0 +1,391 @@
...
+          }
+        },
+        "tags": [
+          "Disks"
+        ],
+        "operationId": "Disks_Get",
+        "description": "Returns the disk.",
+        "parameters": [
+          {
+            "$ref": "../../preview/2015-12-01-preview/Compute.json#/parameters/SubscriptionIdParameter"
```
Comment (bganapa) [inline specification/azsadmin/resource-manager/compute/Microsoft.Compute.Admin/stable/2021-04-01/Disks.json:90]: Its better to copy the SubscriptionIdParameter and LocationParameter to this  file itself insted of cross referencing it from a 2015 preview version

Comment (FireDefend) [reply specification/azsadmin/resource-manager/compute/Microsoft.Compute.Admin/stable/2021-04-01/Disks.json]: removed reference

File: specification/azsadmin/resource-manager/commerce/Microsoft.Commerce.Admin/preview/2015-06-01-preview/Commerce.json (hunk 1 above)
Comment (FireDefend) [reply specification/azsadmin/resource-manager/commerce/Microsoft.Commerce.Admin/preview/2015-06-01-preview/Commerce.json]: reverted

**Comment by FireDefend** [comment]
@bganapa Hi Bala, hope you are doing well. Any updates for this PR?

**Comment by bganapa** [comment]
@FireDefend there are swagger validations errors, could you please look in to them? Can you pls also ping me at bganapa internally?

**Comment by bganapa** [comment]
@leni-msft  pls do the needful for getting this reviewed by ARM team

**Comment by markcowl** [comment]
@FireDefend  Please fill out the checklist thoroughly.  No review can proceed until you have read and filled out the checklist.

**Comment by bganapa** [comment]
@NetStorm-E  please ping me at bganapa. The commits from @TheOnlyWei  should be omitted from your PR. Those commits are in this PR
https://github.com/Azure/azure-rest-api-specs/pull/13283

**Comment by bganapa** [comment]
@FireDefend  Can you pls look at the cross version breaking changes and make sure that are expected? This seems to be anew thing in the PR validation

**Comment by FireDefend** [comment]
> @FireDefend Can you pls look at the cross version breaking changes and make sure that are expected? This seems to be anew thing in the PR validation

Hi Bala, I have checked, those changes are expected.

**Comment by FireDefend** [comment]
closed cause another same PR got merged
https://github.com/Azure/azure-rest-api-specs/pull/14426

**Pull request closed without merging**